orbit/models/ets.py

```diff
@@ -4,25 +4,18 @@
 from copy import copy, deepcopy
 
 from ..constants import ets as constants
-from ..constants.constants import (
-    PredictMethod
-)
-from ..estimators.stan_estimator import StanEstimatorMCMC, StanEstimatorVI, StanEstimatorMAP
-from ..estimators.pyro_estimator import PyroEstimatorVI, PyroEstimatorMAP
+from ..constants.constants import PredictMethod
+from ..estimators.stan_estimator import StanEstimatorMCMC, StanEstimatorMAP, StanEstimatorVI
 from ..exceptions import IllegalArgument, ModelException, PredictionException
 from ..initializer.ets import ETSInitializer
-from .base_model import BaseModel
+from .module import BaseModule, FullBayesianModule, AggregatedPosteriorModule, MAPModule
```

Comment:
just a comment: can we rename `module` to `base_module` since module is a general term in python. if you agree, we need to do changes according in other importing parts.